qa/tasks/cephfs/test_mirroring.py

```diff
@@ -421,6 +421,34 @@ def test_directory_commands(self):
         self.disable_mirroring(self.primary_fs_name, self.primary_fs_id)
         self.mount_a.run_shell(["rmdir", "d1"])
 
+    def test_directory_command_ls(self):
+        dir1 = '/d1'
+        dir2 = '/d2'
+        self.mount_a.run_shell(["mkdir", dir1])
+        self.mount_a.run_shell(["mkdir", dir2])
+        self.enable_mirroring(self.primary_fs_name, self.primary_fs_id)
+        try:
+            self.add_directory(self.primary_fs_name, self.primary_fs_id, dir1)
```

Comment:
The intention looks weird. @joscollin please check that too.

Reply:
@vshankar 
`add_directory` is necessary here. Then only `ceph fs snapshot mirror ls` list the directories. Otherwise, it's empty: `[]`

Reply:
That's fine. My comment was for the code indentation, which looked a bit weird it Github UI.

Reply:
There was a typo https://github.com/ceph/ceph/pull/58966#discussion_r1734695302. I'll check the indentation and fix it.

Reply:
@vshankar I've checked the code and there's no indentation issues found. I'm posting the whitespace-mode of `test_directory_command_ls` below for reference.

![image](https://github.com/user-attachments/assets/643ee159-aea5-4af7-999a-4c0d7e888a01)